recipe/meta.yaml

```diff
@@ -14,14 +14,16 @@ source:
 build:
   number: 0
   script: {{ PYTHON }} -m pip install . --no-deps -vv
-  noarch: python
+  skip: True  # [py<36]
 
 requirements:
   host:
     - python
     - pip
+    - setuptools
```

Comment:
Should be `setuptools >=40.6.0`, see https://github.com/mvantellingen/python-zeep/blob/1e55eeb6b0b992e90234296b1a1392253528dfef/pyproject.toml#L2

Reply:
added